test/integration/containerized/test_container_management.py

```diff
@@ -94,26 +94,37 @@ def test_invalid_registry_host(tmp_path, runtime):
             'container_image': image_name,
             'container_options': ['--user=root', '--pull=always'],
         },
-        container_auth_data={'host': 'https://somedomain.invalid', 'username': 'foouser', 'password': '349sk34'},
+        container_auth_data={'host': 'somedomain.invalid', 'username': 'foouser', 'password': '349sk34', 'verify_ssl': False},
         ident='awx_123'
     )
     assert res.status == 'failed'
     assert res.rc > 0
+    assert os.path.exists(res.config.registry_auth_path)
 
     result_stdout = res.stdout.read()
     if runtime == 'podman':
         assert image_name in result_stdout
         assert 'unauthorized' in result_stdout
+        auth_file_path = res.config.registry_auth_path
+        registry_conf = os.path.join(os.path.dirname(res.config.registry_auth_path), 'registries.conf')
     else:
         assert 'access to the requested resource is not authorized' in result_stdout
+        auth_file_path = os.path.join(res.config.registry_auth_path, 'config.json')
+        registry_conf = os.path.join(res.config.registry_auth_path, 'registries.conf')
```

Comment:
The `else` isn't strictly necessary.
```suggestion
    auth_file_path = os.path.join(res.config.registry_auth_path, 'config.json')
    registry_conf = os.path.join(res.config.registry_auth_path, 'registries.conf')
    error_msg = 'access to the requested resource is not authorized'
    if runtime == 'podman':
        assert image_name in result_stdout
        error_msg = 'unauthorized'
        auth_file_path = res.config.registry_auth_path
        registry_conf = os.path.join(os.path.dirname(res.config.registry_auth_path), 'registries.conf')
        
    assert error_msg in result_stdout

```